documents/operating.md

```diff
@@ -0,0 +1,216 @@
+Bad Idea Factory Limited Liability Corporation
+===================================================================
+
+Contents:
+
+1.  Official Operating Agreement
+2.  Partnership
+3.  Projects
+4.  Money
+5.  Governing Process
+6.  Policies
+
+---------------------------------------------------------------------
+SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
+---------------------------------------------------------------------
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?)
```

Comment:
There is some language later in the document about removing CO's / resigning as a CO -- Lets address your concern for sure, but I'd rather see us change the rules for quorum or better streamline that process to prevent unnecessary explosion than remove the requirement to hold functional meetings.

Keep in mind that votes are needed for us to be able to do a lot of important things as an org (adding members, adding projects, dealing with $$).  It seems to me that a full year of not holding votes indicates serious dysfunction or apathy or both -- to the point that the company has failed.  

This has external issues as well -- for example: lets say we have only two people who are still paying attention, the company has no money to pay tax fees, and no ability to vote on approving a capital contribution to cover those taxes because we can't hold quorum.  That'd be no good.

Also FWIW this is the kind of thing that our lawyer will surely be able to give good insight on -- it's exactly the kind of "keep your client from shooting themselves in the foot even though they're idiots" kind of thing lawyers live for!